indigo_app/static/javascript/indigo/views/document_editor.js

```diff
@@ -657,6 +687,25 @@
       return (!this.sourceEditor.editing || confirm("You will lose your changes, are you sure?"));
     },
 
+    saveEditTimes: function() {
+      var sourceEditor = this.sourceEditor;
+      fetch(`/api/documents/${Indigo.Preloads.document.id}/activity/edits`, {
+        method: 'POST',
+        headers: {
+          "Content-Type": "application/json",
+          "X-CSRFToken": Indigo.csrfToken
+        },
+        body: JSON.stringify(sourceEditor.editTimes)
+      }).then(function(response) {
+        if(response.ok) {
+          sourceEditor.editTimes = [];
+        } else {
+          console.log('Response data from save edit times request', response);
+          throw new Error('Could not save time spent on this document');
```

Comment:
sentry will pick this up, and we can see what is in the console for debugging purposes